### PR TITLE
Hg 624 Enable performance monitoring

### PR DESCRIPTION
**Pull request by kenkouot**
My previous attempts to enable performance monitoring failed do to use of a custom environment variable that wasn't available at **build time**. I've simplified this by simply binding to the environment instead.

### CODE REVIEW

File: server/views/_layouts/ember-main.hbs [hunk 1]
```diff
@@ -90,7 +90,7 @@
 			<script src="/front/vendor/ember-hammer/ember-hammer.js"></script>
 			<script src="/front/vendor/i18next/i18next.js"></script>
 			<script src="/front/vendor/vignette/dist/vignette.js"></script>
-			<!-- @ifdef ENABLE_WEPPY -->
+			<!-- @if WIKIA_ENVIRONMENT!='dev' -->
```
Comment (rogatty) [inline server/views/_layouts/ember-main.hbs]: Let's keep it consistent, so `<!-- @if WIKIA_ENVIRONMENT='prod' -->`

**Comment by lizlux** [comment]
LGTM. Do we need to remove the `ENABLE_WEPPY` flag from any other places that might be setting it?

**Comment by rogatty** [comment]
LGTM
@lizlux: it's already been done in https://github.com/Wikia/chef-repo/pull/5052

**Comment by lizlux** [comment]
:+1: thanks @rogatty!

**Pull request merged**